### ISSUE TITLE
Feature Request: Remove/Disable the Structure/Support system on buildings.

### ISSUE THREAD

**Issue by DaNnySaNn8391** [valheimPlus/ValheimPlus#32]
Would it be possible to add the ability to disable structural weakness on buildings.

**Comment by nxPublic** [comment]
Implemented in 0.8.5.

```ini

[StructuralIntegrity]

; Change false to true to enable this section
enabled=false

; Disables the entire structural integrity system and allows for placement in free air, does not prevent building damage.
disableStructuralIntegrity=false;

; Each of these values reduce the loss of structural integrity by % less. The value 100 would result in disabled structural integrity and allow placement in free air.
wood=0
stone=0
iron=0
hardWood=0

```

**Comment by DaNnySaNn8391** [comment]
Awesome guys thank you very much, keep up the great work <3
